jetty-server/src/main/java/org/eclipse/jetty/server/session/AbstractSessionCache.java

```diff
@@ -135,7 +136,7 @@
      * @param newValue the new value
      * @return true if replacement was done
      */
-    public abstract boolean doReplace(String id, Session oldValue, Session newValue);
+    protected abstract boolean doReplace(String id, Session oldValue, Session newValue);
```

Comment:
changing visibility in minor release? (maybe in 10?)

Reply:
As per previous comment.